frontend/package.json

```diff
@@ -18,6 +18,7 @@
     "test-gui": "yarn run test-suite --suite all",
     "test-kubevirt": "yarn run test-suite --suite kubevirt --params.openshift true",
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
+    "webdriver-update": "./node_modules/.bin/webdriver-manager update",
```

Comment:
There is already `webdriver-update` script a few lines above.
Can we stay synced with upstream on that?